src/pages/chats/[topic].tsx

```diff
@@ -1,46 +1,69 @@
-import { getTopicData, isSupportedTopic, Topic } from '@/constants/topics'
+import { CHAT_PER_PAGE } from '@/constants/chat'
 import ChatPage from '@/modules/_chats/ChatPage'
+import { getPostQuery } from '@/services/api/query'
 import { getCommentIdsQueryKey } from '@/services/subsocial/commentIds'
-import { getPostQuery } from '@/services/subsocial/posts'
 import { getSubsocialApi } from '@/subsocial-query/subsocial'
-import { getCommonServerSideProps } from '@/utils/page'
+import { getSpaceId } from '@/utils/env/client'
+import { getCommonStaticProps } from '@/utils/page'
+import { createPostSlug, getPostIdFromSlug } from '@subsocial/utils/slugify'
 import { dehydrate, QueryClient } from '@tanstack/react-query'
+import { GetStaticPaths } from 'next'
+import { getPostsFromCache } from '../api/posts'
 
-export const getServerSideProps = getCommonServerSideProps<{
+export const getStaticPaths: GetStaticPaths = async () => {
```

Comment:
what do you mean by this?
do you mean if we add new post to a space, it should show up in the list?

Reply:
No, when I will create new post in new space and put it to Grill it shoud open chat for the post, but we wont show it in list

Reply:
you can already do that, thats the behavior of fallback blocking

so for example, you can go to

staging-grillchat.subsocial.network/chats/eth-6

well, because the post dont have any reply or content it wont show anything though